Xamarin.Forms.Platform.iOS/Renderers/TimePickerRenderer.cs

```diff
@@ -195,7 +197,35 @@ void UpdateCharacterSpacing()
 		void UpdateTime()
 		{
 			_picker.Date = new DateTime(1, 1, 1).Add(Element.Time).ToNSDate();
-			Control.Text = DateTime.Today.Add(Element.Time).ToString(Element.Format);
+			if (Element.Format.Equals(""))
+			{
+				string iOSLocale = NSLocale.CurrentLocale.CountryCode;
+				var cultureInfos = CultureInfo.GetCultures(CultureTypes.AllCultures)
+							  .Where(c => c.Name.EndsWith("-" + iOSLocale));
```

Comment:
can you just change this code to 
```C#
var cultureInfo = CultureInfo.GetCultures(CultureTypes.AllCultures)
							  .Where(c => c.Name.EndsWith("-" + iOSLocale)).First();
```

It doesn't look like you are using the list of `cultureInfos` but always just the first element after this.
It also looks like you could move this search above the if statement because it's doing the same search inside the else as it is here.

As a side note about `Linq` (i.e. `.Where`)

`Linq` operators don't perform the given operation against the provided list until you access the data so every time this code calls `cultureInfos.First()` it's iterating through the data again and again.

It's best to just save off the data you need as early as you can and then operate against that.

You probably don't even need `linq` you could just foreach over `cultureInfos` and store that result into a local variable and then use that.